src/applications/appeals/995/components/EvidenceRecords.jsx

```diff
@@ -0,0 +1,352 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+
+import {
+  VaTextInput,
+  VaSelect,
+  VaCheckboxGroup,
+  VaMemorableDate,
+  VaModal,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+
+import { countries, states } from 'platform/forms/address';
+import FormNavButtons from 'platform/forms-system/src/js/components/FormNavButtons';
+// import { FormNavButtons } from '@department-of-veterans-affairs/platform-forms-system';
```

Comment:
Do you still need these comments?

Reply:
Yeah, leave it... it's a reminder that we need to figure out the `@department...` imports. Someday.